File: src/main/kotlin/net/wavem/uvc/rms/gateway/event/domain/event_info/EventInfo.kt
package net.wavem.uvc.rms.gateway.event.domain.event_info

import net.wavem.uvc.rms.gateway.location.domain.position.LocationPosition

data class EventInfo(
    val eventId : String?,
    val eventCd : String?,
    val eventSubCd : String?,
    val areaClsf : String?,
    val floor : String?,
    val batteryLevel : Int?,
    val location : LocationPosition?
) {
}

File: src/rms/response/event/domain/eventInfo/event_info.py
from rms.response.event.domain.eventInfo.location.event_info_location import EventInfoLocation


class EventInfo():
    def __init__(
        self, 
        eventId: str = None,
        eventCd: str = None,
        eventSubCd: str = None,
        areaClsf: str = None,
        floor: str = None,
        batteryLevel: int = None,
        location: dict = None
    ) -> None:
        self.eventId = eventId
        self.eventCd = eventCd
        self.eventSubCd = eventSubCd
        self.areaClsf = areaClsf
        self.floor = floor
        self.batteryLevel = batteryLevel
        self.location = location